x/stakeibc/keeper/host_zone_test.go

```diff
@@ -248,4 +251,11 @@ func (s *KeeperTestSuite) TestAddDelegationToValidator() {
 	// Attempt to modify a validator that doesn't exist - it should fail
 	err = s.App.StakeibcKeeper.AddDelegationToValidator(s.Ctx, hostZone, "does_not_exist", sdkmath.NewInt(1000), "")
 	s.Require().ErrorContains(err, "validator not found")
+
+	// Attempt to subtract more than the total delegations on the host - it should fail
+	// Here, w set the validator's delegation to be much higher than the TotalDelegation
+	//   (which should not be possible in practice)
+	hostZone.Validators[updatedIndex].Delegation = sdkmath.NewInt(10000)
```

Comment:
Isn't this an impossible state to reach, where the TotalDelegations and sum of Delegations are out of sync

Reply:
yeah see previous comment

Reply:
fair enough, can't hurt to keep the extra safety check in